index.js

```diff
@@ -2,66 +2,66 @@
   const canvas = document.getElementById("canvas");
   const context = canvas.getContext('2d');
   let gameSpeed = 30;
-  let computerPaddleSpeed = 5;
-  const MAX_SCORE = 3;
-  let playerScore = 0;
-  let computerScore = 0;
-
+  let paddleSpeed = 5;
 
+  class ObjXY {
+    constructor(x, y) {
+      this.x = x;
+      this.y = y;
+    }
+  }
 
   /**
-   * Ball Constructor
+   * Create instance of a ball
    */
-  function Ball() {
-    this.color = "white";
-    this.size = {
-      x: 10,
-      y: 10
+  class Ball {
+    constructor() {
+      this.color = "white";
+      this.size = new ObjXY(10, 10);
+      this.position = new ObjXY(
+        (canvas.width / 2) - (this.size.x / 2),
+        (canvas.height / 2) - (this.size.y / 2)
+      );
+      this.velocityX = -10;
+      this.velocityY = 10;
     };
-    this.position = {
-      x: (canvas.width / 2) - (this.size.x / 2),
-      y: (canvas.height / 2) - (this.size.y / 2)
-    };
-    this.velocityX = -10;
-    this.velocityY = 10;
-  };
 
-  Ball.prototype.calculatePosition = function () {
-    this.position.x = this.position.x + this.velocityX;
-    this.position.y = this.position.y + this.velocityY;
-  };
+    calculatePosition = () => {
+      this.position.x = this.position.x + this.velocityX;
+      this.position.y = this.position.y + this.velocityY;
+    }
+  }
 
   /**
    * Paddle constructor
    * @param {string} position Accepts left or right 
+   * @param {number}}paddleSpeed speed at which paddle can react
    */
-  function Paddle(position, paddleSpeed) {
-    this.color = "white";
-    this.size = {
-      x: 15,
-      y: 80
-    };
-    this.position = {
-      x: position === "left" ? 10 : canvas.width - 25,
-      y: canvas.height / 2
-    };
-    this.speed = paddleSpeed
-  };
-
-  /**
-   * @param {object} ball
-   */
-  Paddle.prototype.HandleComputerMovement = function (ball) {
-    const topOfPaddle = this.position.y;
-    const bottomOfPaddle = this.position.y + this.size.y;
-
-    if (topOfPaddle > ball.position.y && bottomOfPaddle > ball.position.y) {
-      this.position.y = this.position.y - this.speed
-    } else if (topOfPaddle < ball.position.y && bottomOfPaddle < ball.position.y) {
-      this.position.y = this.position.y + this.speed
+  class Paddle {
+    constructor(position, paddleSpeed) {
+      this.color = "white";
+      this.size = new ObjXY(15, 80);
+      this.position = new ObjXY(
+        (position === "left" ? 10 : canvas.width - 25),
+        (canvas.height / 2)
+      );
+      this.speed = paddleSpeed;
     }
 
-  }
+    /**
+     * @param {object} ball
+     */
+    HandleComputerMovement = (ball) => {
```

Comment:
I dont think you start function names with a capital
```suggestion
    handleComputerMovement = (ball) => {
```